zcash_primitives/src/transaction/builder.rs

```diff
@@ -1281,7 +1318,7 @@ mod tests {
         let mut witness1 = IncrementalWitness::from_tree(tree.clone()).unwrap();
 
         // Fail if there is insufficient input
-        // 0.0003 z-ZEC out, 0.0002 t-ZEC out, 0.0001 t-ZEC fee, 0.00059999 z-ZEC in
+        // 0.0003 z-ZEC out, 0.00015 t-ZEC out, 0.0001 t-ZEC fee, 0.00059999 z-ZEC in
```

Comment:
I believe this is incorrect (pre-existing error):
```suggestion
        // 0.0003 z-ZEC out, 0.00015 t-ZEC out, 0.00015 t-ZEC fee, 0.00059999 z-ZEC in
```
There are three logical actions in this tx: two for Sapling (max of two spends after padding, and one output), and one for the transparent output.